graphql/types/SponsoredProduct.graphql

```diff
@@ -0,0 +1,74 @@
+type SponsoredProduct {
+  """
+  Unique identifier of the product. Can be SKU, EAN or VTEX Product ID.
+  """
+  identifier: ProductUniqueIdentifier
+
+  """
+  ID of the sponsored product.
+  """
+  productId: ID @deprecated(reason: "Use `identifier` instead.")
```

Comment:
I've added this `@deprecated` directive. It won't break the relevanC schema, but it should raise a warning.